docker-compose/start_dse_68.sh

```diff
@@ -47,4 +47,11 @@ export JSONIMAGE
 
 echo "Running with DSE $DSETAG, Stargate $SGTAG, JSON API $JSONIMAGE:$JSONTAG"
 
-docker-compose up -d
+COMPOSE_ARGS=("-d")
```

Comment:
Adding the `--wait` option to docker compose (if available). Lack of this option appears to be the cause of the failure in this workflow: https://github.com/stargate/jsonapi/actions/runs/4494697370. The schema creation command(s) failed due to the JSON API not being all the way up, causing following calls in the NB workload to fail. (This parallels errors seen in automated testing of Postman scripts against Docker compose-based configuration when `--wait` flag was not used.)

Reply:
OK cool..